src/Interface.java

```diff
@@ -31,37 +29,54 @@ public void run() {
 
 		frame.pack();
 		frame.setVisible(true);
-
 	}
 
+	// Method creating components - design of interface
 	private void createComponents(Container container) {
 
 		GridLayout layout = new GridLayout(1, 3);
 		container.setLayout(layout);
-		dropDown = new JComboBox(new String[] { "Montana", "Say Hello" });
-
-		dropDown.addActionListener(new ActionListener() {
-
-			@Override
-			public void actionPerformed(ActionEvent e) {
-
-			}
-		});
-
-		JTextArea textAreaLeft = new JTextArea("");
-		JTextArea textAreaRight = new JTextArea();
-		JButton copyButton = new JButton("Play");
-
-		ButtonListener play = new ButtonListener(textAreaLeft, textAreaRight);
-		copyButton.addActionListener(play);
-
-		container.add(copyButton);
```

Comment:
I changed layout a little bit. I added a "stop" button to better control player's functionality. I deleted JTextArea - it is unnecessary. I changed the name "copyButton" to "playButton". I added strings with filepaths and boolean to check if dropdown menu is changed. I deleted ActionListener from here and moved it to MessageListener (and added code in it).